packages/tldraw/src/lib/ui.css

```diff
@@ -264,6 +264,16 @@
 	opacity: 1;
 }
 
+.tlui-layout__mobile .tlui-button__tool {
+	height: 48px;
+	width: 44px;
+}
+
+.tlui-layout__mobile .tlui-button__tool > .tlui-icon {
+	height: 16px;
+	width: 16px;
+}
+
```

Comment:
We already dynamically apply the `tlui-layout__mobile` class, so let's use that instead of a media query.